build/templates/_converters.py.mako

```diff
@@ -254,6 +254,16 @@ def _convert_to_bytes(value):  # noqa: F811
     pass
 
 
+<%
+# This converter is only needed by nidigital
+%>\
+% if config['module_name'] == 'nidigital':
+# nidigital specific converter
+def convert_comma_separated_string_to_list(comma_separated_string):
```

Comment:
Make sure you do a full codegen. If you modify this file, then the generated flavor for all drivers should appear in the diff!

Reply:
done